docs/source/en/api/pipelines/kandinsky.mdx

```diff
@@ -212,9 +212,9 @@ init_image = load_image(
     "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main" "/kandinsky/cat.png"
 )
 
-mask = np.ones((768, 768), dtype=np.float32)
+mask = np.zeros((768, 768), dtype=np.float32)
 # Let's mask out an area above the cat's head
-mask[:250, 250:-250] = 0
+mask[:250, 250:-250] = 1
```

Comment:
Let's maybe add a "Tip warning" here to ensure the users are communicated about this through and through? 

Maybe something like:

```
<Tip warning=true>

Note that the above change was introduced in the following pull request: https://github.com/huggingface/diffusers/pull/4207. So, if you're using a source installation of `diffusers` or the latest release, you should upgrade your inpainting code to follow the above. 

</Tip>
```

WDYT?

Reply:
Also the comment in the PR description is wrong (just in case we want to copy/paste it somewhere)